src/apps/vpn/ui/screens/settings/ViewLanguage.qml

```diff
@@ -11,152 +11,162 @@ import components 0.1
+                                      obj.localizedLanguageName.toLowerCase().includes(filterValue);
+                                  }
+            _searchBarHasError: repeater.count === 0
+            _searchBarPlaceholderText: MZI18n.LanguageViewSearchPlaceholder
+
+        }
+
+        ColumnLayout {
+            spacing: 0
+            visible: searchBar.isEmpty
+
+            MZRadioDelegate {
+                id: systemLanguageRadioButton
+
+                Layout.fillWidth: true
+                Layout.leftMargin: MZTheme.theme.windowMargin
+                Layout.rightMargin: MZTheme.theme.windowMargin
+
+                radioButtonLabelText: MZI18n.LanguageViewSystemLanguageButtonTitle
+                checked: MZSettings.languageCode === ""
+                activeFocusOnTab: true
+                onClicked: {
+                    MZSettings.languageCode = ""
+                }
+
+                Keys.onDownPressed: if(repeater.itemAt(0)) repeater.itemAt(0).pushFocusToRadio()
+
+                accessibleName: `${radioButtonLabelText}. ${systemLanguageRadioButtonDescription.text}`
+            }
+
+            MZTextBlock {
+                id: systemLanguageRadioButtonDescription
+
+                Layout.fillWidth: true
+                Layout.topMargin: 4
+                Layout.leftMargin: MZTheme.theme.windowMargin + systemLanguageRadioButton.labelX
+                Layout.rightMargin: MZTheme.theme.windowMargin
+
+                text: MZI18n.LanguageViewSystemLanguageButtonDescription
             }
 
-            Repeater {
-                id: repeater
-
-                model: searchBar.getProxyModel()
-
-                Component.onCompleted: {
-                    for (let idx = 0; idx < repeater.count; idx++) {
-                        const langItem = repeater.itemAt(idx);
-                        if (langItem.isSelectedLanguage) {
-                            const yCenter = vpnFlickable.height / 2;
-                            const selectedItemYPosition = langItem.y + (languageList.y + repeater.y + MZTheme.theme.menuHeight + toggleCard.height + langItem.height) - yCenter;
-                            const destinationY = (selectedItemYPosition + vpnFlickable.height >col.implicitHeight) ? col.implicitHeight - vpnFlickable.height / 2: selectedItemYPosition;
-                            if (destinationY < 0) {
-                                return;
-                            }
-                            setContentY(destinationY)
+            Rectangle {                
+                Layout.preferredHeight: 1
+                Layout.fillWidth: true
+                Layout.topMargin: MZTheme.theme.vSpacing
+                Layout.leftMargin: 18.5
+                Layout.rightMargin: 18.5
+
+                color: "#E7E7E7"
+            }
+        }
+
+        Repeater {
+            id: repeater
+
+            model: searchBar.getProxyModel()
+
+            Component.onCompleted: {
```

Comment:
Running locally the selected language is not centered but this may already be broken and not caused by this PR. @peterbenvenuto do we care about centering the selected language when the view opens?

Reply:
It's not something we currently do for the languages view, but could be a good opportunity to add it!